README.md

```diff
@@ -218,15 +220,23 @@ mirror database content from a designated master. This replication scheme allows
 us to sync databases. However a `replicant` is only for read-only transaction, thus 
 we can't write new data to it. The whole database cluster will be replicated.
 
+### Database permissions
+Since we are using a role ${REPLICATION_USER}, we need to ensure that he has access to all 
+the tables in a particular schema. So if a user adds another schema called `data`
+to the database `gis` he also has to update the permission for the user
+with the following SQL assuming the ${REPLICATION_USER} is called replicator
+
+     ALTER DEFAULT PRIVILEGES IN SCHEMA data GRANT SELECT ON TABLES TO replicator;
+
```

Comment:
why? The replication role is JUST for replication and not for using the slave. Normal users will use the slave. So the replication role won't be doing select statements.

Reply:
Without giving select permissions the replication user was failing to replicate anything and the error was `No permission for the required table` I guess I was connecting to the slave using the replicator role. I will try it as a normal user and see if it works without altering the permissions